### DISCUSSION TITLE
Default commentary language for all streams

### DISCUSSION THREAD

**Discussion by stefanverkerknl** [f1multiviewer/issue-tracker#115] [Feature requests] [answered]
Please consider adding a feature which enables the user to set a default commentary language in the options menu, so that non-saved video stream layouts get that default language too. Especially handy when you're watching support races which don't have OBC / Data Channel / Driver Tracker etc.

Thanks!

**Comment by spambas** [comment]
Same here, need to change the language to English every single time when I open anything other then the saved setup.
Getting a bit annoying...

**Comment by f1multiviewer** [comment, marked as answer]
This has been fixed in the latest release!
